arm-sql/2014-04-01/swagger/serverDisasterRecoveryConfiguration.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Sql Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration": {
```

Comment:
Discussed with SDK team and the implementing team; we all agree that the response returning a list of one or 404 is a bug.  Going to model this as a typical List function and let the client-side users deal handle the bug how they wish.

---
In reply to: [83980199](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r83980199) [](ancestors = 83980199,83757155,83288385,83281234)